Makefile

```diff
@@ -40,6 +46,9 @@ worker:
 cli:
```

Comment:
Shouldn't we also add `make` commands for other useful macros, e.g. `make console-production`/`make start-production`/`make stop-production`/...? Is there a better way to handle that? Maybe just [an argument](http://stackoverflow.com/questions/2826029/passing-additional-variables-from-command-line-to-make) provided to `Makefile`?